roles/app/files/seeds.rb

```diff
@@ -1,29 +0,0 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-require 'yaml'
-require 'csv'
-
-# -- Spree
-unless Spree::Country.find_by_iso(ENV['DEFAULT_COUNTRY_CODE'])
```

Comment:
this script feels a bit more decoupled from Australia, so I might bring it to the app as well